cmd/update-main.go

```diff
@@ -100,7 +100,7 @@ func GetCurrentReleaseTime() (releaseTime time.Time, err error) {
 
 func isDocker(cgroupFile string) (bool, error) {
 	cgroup, err := ioutil.ReadFile(cgroupFile)
-	if os.IsNotExist(err) {
+	if os.IsNotExist(err) || os.IsPermission(err) {
```

Comment:
Do you mean in any other container system `/proc/self/cgroup` might not be permission to read the file?